tests/functional/api.php

```diff
@@ -0,0 +1,93 @@
+];
+
+$number = [
+    ['?action=coverage&locales[]=it&locales[]=fr', 200, '1.92'],
+    ['?action=coverage&locales[]=fr', 200, '0.32'],
+    ['?action=coverage&locales[]=foo', 200, '0.00'],
+];
+
+$obj = new \pchevrel\Verif('Check API HTTP responses');
+$obj->setHost('localhost:8083');
+
+$check = function ($object, $paths, $type) {
+        foreach ($paths as $values) {
+            list($path, $http_code, $content) = $values;
+            switch ($type) {
+                case 'json':
+                    $object
+                        ->setPath($path)
+                        ->fetchContent()
+                        ->hasResponseCode($http_code)
+                        ->isJson()
+                        ->isEqualTo($content);
+                break;
+                case 'number':
+                    $object
+                        ->setPath($path)
+                        ->fetchContent()
+                        ->hasResponseCode($http_code)
+                        ->isNumeric()
+                        ->isEqualTo($content);
+                break;
+                default:
+                    $object
+                        ->setPath($path)
+                        ->fetchContent()
+                        ->hasResponseCode($http_code)
+                        ->isEqualTo($content);
+                break;
```

Comment:
There is a lot of method duplication, this would be probably simpler:
```php
$check = function ($paths, $type) use ($obj) {
        foreach ($paths as $values) {
            list($path, $http_code, $content) = $values;
            $obj
                ->setPath($path)
                ->fetchContent()
                ->hasResponseCode($http_code)
                ->isEqualTo($content);

            switch ($type) {
                case 'json':
                    $obj->isJson();
                    break;
                case 'number':
                    $obj->isNumeric();
                    break;
                default:
                    break;
            }
        }
};
```
Also please indent break statements:
https://github.com/php-fig/fig-standards/blob/master/accepted/fr/PSR-2-coding-style-guide.md#52-switch-case